api_tests/users/views/test_user_detail.py

```diff
@@ -1330,13 +1330,13 @@ def test_user_put_profile_date_validate_end_month_dependency(self, app, user_one
         # No endMonth with endYear
         res = app.put_json_api(user_one_url, end_month_dependency_payload, auth=user_one.auth, expect_errors=True)
         assert res.status_code == 400
-        assert res.json['errors'][0]['detail'] == "u'endYear' is a dependency of u'endMonth'"
+        assert res.json['errors'][0]['detail'] == "'endYear' is a dependency of 'endMonth'"
 
     def test_user_put_profile_date_validate_start_month_dependency(self, app, user_one, user_one_url, start_month_dependency_payload):
         # No endMonth with endYear
         res = app.put_json_api(user_one_url, start_month_dependency_payload, auth=user_one.auth, expect_errors=True)
         assert res.status_code == 400
-        assert res.json['errors'][0]['detail'] == "u'startYear' is a dependency of u'endYear'"
+        assert res.json['errors'][0]['detail'] == "'startYear' is a dependency of 'startMonth'"
```

Comment:
These are in fact both dependencies of start yeah but only produce one error, I think just because it doesn't return multiple dependency violations as multiple errors. So this changed but still one error is returned in the json.